terraform/cloud-platform-components/templates/prometheus-operator.yaml.tpl

```diff
@@ -465,10 +465,13 @@ grafana:
     GF_USERS_ALLOW_SIGN_UP: "false"
     GF_USERS_AUTO_ASSIGN_ORG_ROLE: "Viewer"
     GF_USERS_VIEWERS_CAN_EDIT: "true"
-    GF_AUTH_GITHUB_ENABLED: "true"
-    GF_AUTH_GITHUB_ALLOW_SIGN_UP: "true"
     GF_SMTP_ENABLED: "false"
-    GF_AUTH_GITHUB_ALLOWED_ORGANIZATIONS: "ministryofjustice"
+    GF_AUTH_GENERIC_OAUTH_ENABLED: "true"
+    GF_AUTH_GENERIC_OAUTH_ALLOW_SIGN_UP: "true"
+    GF_AUTH_GENERIC_OAUTH_TEAM_IDS:
```

Comment:
This and the following value are empty, do we need them here?

Reply:
Looking at what other people have done, they have let them blank.
I will test it without the values.